test/cloudant.test.js

```diff
@@ -47,9 +47,13 @@ describe('cloudant connector', function() {
         ],
       },
     });
-    Product.destroyAll(function(err) {
-      CustomerSimple.destroyAll(function(err) {
-        done();
+
+    db.automigrate(function(err) {
```

Comment:
sounds good, I prefer to update all of them in https://github.com/strongloop/loopback-connector-cloudant/pull/91, it is a test case clean up pr.